actions/fileSys.go

```diff
@@ -0,0 +1,146 @@
+package actions
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	sys "github.com/arschles/sys"
+	"github.com/deis/deisrel/testutil"
+)
+
+type fileSys interface {
+	sys.FS
+	Create(string) (*os.File, error)
+	Stat(string) (os.FileInfo, error)
+	MkdirAll(string, os.FileMode) error
+	WriteFile(string, []byte, os.FileMode) error
+}
+
+func getRealFileSys() fileSys {
+	return &realFileSys{}
+}
+
+type realFileSys struct {
+	fileSys
```

Comment:
@vdice you don't need this in the struct - as long as the struct has all of `fileSys`'s funcs on it, it will implement the interface